follower/consensus_follower.go

```diff
@@ -36,8 +37,8 @@ type Config struct {
 	networkPrivKey   crypto.PrivateKey   // the network private key of this node
 	bootstrapNodes   []BootstrapNodeInfo // the bootstrap nodes to use
 	bindAddr         string              // address to bind on
-	db               *badger.DB          // the badger DB storage to use for the protocol state
-	dataDir          string              // directory to store the protocol state (if the badger storage is not provided)
```

Comment:
Support for directly specifying a data directory has been removed. Instead, users must now provide a BadgerDB or PebbleDB instance. To create a PebbleDB instance, use the InitPebbleDB method in the follower/database package.